.github/workflows/run-updatecli.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Install Updatecli in the runner
```

Comment:
I renamed it as otherwise json schema thinks it's an update cli file as it looks for 'updatecli.yaml'